guardian/braze-components
# Move guardian dependencies to peer dependencies. This reduces bundle size.

## What does this change?
Move @guardian libs from dependencies into peer dependencies. This reduces bundle size loaded by the user as dotcom-rendering and frontend are loading the libraries anyhow, so before this change we were loading them twice for no good reason. 

## How to test
Run yarn build and check bundle sizes are reduced. yarn link from dotcom rendering and check the components still work. 

## How can we measure success?
Reduced bundle size.

## Have we considered potential risks?
Need to make sure we keep our peerDependencies updated as dotcom rendering and frontend update their libraries.

No UI changes.
